tests/test_phase_psf.py

```diff
@@ -52,8 +53,51 @@ def test_aperture():
 
 
 @timer
+def test_atm_screen_size():
+    """Test for consistent AtmosphericScreen size and scale."""
+    screen_size = 10.0
+    screen_scale = 0.1
+    atm = galsim.AtmosphericScreen(screen_size=screen_size, screen_scale=screen_scale)
+    # AtmosphericScreen will preserve screen_scale, but will adjust screen_size as necessary to get
+    # a good FFT size.
+    assert atm.screen_scale == screen_scale
+    assert atm.screen_size != screen_size
+    np.testing.assert_equal(atm.screen_size, atm.npix * atm.screen_scale,
+                            "Inconsistent atmospheric screen size and scale.")
+
+
+@timer
+def test_structure_function():
+    """Test that AtmosphericScreen generates approximately the right structure function for infinite
+    outer scale.
+    """
+    rng = galsim.BaseDeviate(4815162342)
+    r0_500 = 0.2
+    L0 = None
+    screen_scale = 0.05
+    screen_size = 100.0
+
+    # Theoretical pure Kolmogorov structure function (at 500 nm!):
+    D_kolm = lambda r: 6.8839 * (r/r0_500)**(5./3)
+
+    atm = galsim.AtmosphericScreen(screen_size=screen_size, screen_scale=screen_scale,
+                                   r0_500=r0_500, L0=L0, rng=rng)
+    phase = atm.tab2d.table.getVals()[:-1, :-1].copy()
+    phase *= 2 * np.pi / 500.0  # nm -> radians
+    im = galsim.Image(phase, scale=screen_scale)
+    D_sim = galsim.utilities.structure_function(im)
+
+    for r in [0.5, 2.0, 5.0]:  # Only check values far from the screen size and scale.
+        # We're only attempting to verify that we haven't missed a factor of 2 or pi or
+        # something like that here, so set the rtol below to be *very* forgiving.  Since the
+        # structure function varies quite quickly as r**(5./3), this is still a useful test.
+        np.testing.assert_allclose(D_kolm(r), D_sim(r), rtol=0.5,
```

Comment:
Just curious, but what is the error here for these values of r?  Maybe mention the values in a comment to motivate why rtol = 0.5 is necessary.

I'd at least add a print statement here giving D_kolm(r) and D_sim(r) so that people running from the command line (rather than via nosetests) can see the values directly.